website/docs/docs/collaborate/git/version-control-basics.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Version control basics"
+id: version-control-basics
+description: "Leverage git to version control your code."
+sidebar_label: "Version control basics"
+---
+
+# Version control basics
+
+:::info 📌 
+
+To use version control, make sure you are connected to a Git repository in the Cloud IDE. To use version control, make sure you are connected to a Git repository in the IDE.
+
+:::
+
+A [version control](https://en.wikipedia.org/wiki/Version_control) system allows you and your teammates to work collaboratively, safely, and simultaneously on a single project. Version control helps you track all the code changes made in the [Cloud IDE](/docs/develop/develop-in-the-cloud).
+
+When you develop in the Cloud IDE, you can leverage Git directly to version control your code from your browser. This means you can branch, commit, push, and pull code with a couple of clicks - no command line required!
+
+You can create a separate branch to develop and make changes. The changes you make aren’t merged into the main branch unless it successfully passes tests. This helps keep the code organized and improves productivity by making the development process smooth.
+
+![Git overview](https://docs.getdbt.com/img/docs/dbt-cloud/cloud-ide/git-overview.png)
```

Comment:
love this figure! what do you think about moving it over to the /docs/collaborate/git-version-control page?